### DISCUSSION TITLE
Equivalent of `docker run --delete` (starts the VM clean each time)

### DISCUSSION THREAD

**Discussion by mxcl** [cirruslabs/tart#955] [Q&A] [answered]
I generally want a “virgin” state every time I start a VM. `tart delete foo` seems to require I download the image again if I clone to a new one.

**Comment by fkorotkov** [comment, marked as answer]
If you have a remote VM pulled (let's say `tart pull ghcr.io/bar/foo`), then you can do:

```
tart clone ghcr.io/bar/foo foo && tart run foo
```

`tart clone ghcr.io/bar/foo foo` **will override** foo if it exists. Will that work for your use case?

**Reply by mxcl** [reply]
Confirmed this works great thanks. For others: it may be sensible to convert @latest to a sha to prevent updates interfering with debug cycles.
